packages/eas-build-job/src/common.ts

```diff
@@ -16,6 +16,20 @@ export enum ArchiveSourceType {
   PATH = 'PATH',
 }
 
+export enum AndroidBuildResourceClass {
+  ANDROID_DEFAULT = 'ANDROID_DEFAULT',
+  ANDROID_LARGE = 'ANDROID_LARGE',
+}
+
+export enum IosBuildResourceClass {
+  IOS_DEFAULT = 'IOS_DEFAULT',
+  IOS_LARGE = 'IOS_LARGE',
+  IOS_M1 = 'IOS_M1',
+}
+
+export const BuildResourceClass = { ...AndroidBuildResourceClass, ...IosBuildResourceClass };
+export type BuildResourceClass = typeof BuildResourceClass;
```

Comment:
This particular type does not seem necessary. The platform-specific enums are enough.

Reply:
Right now I'm using the union type in eas-cli, e.g. here: https://github.com/expo/eas-cli/pull/1138/files#diff-3373062084e3660ec5409e2867a3a039ca3ecc0eb8536a5ab567878b5f5d667d

I suppose I could refactor it to use `IosBuildResourceClass | AndroidBuildResourceClass` as the type, but the union type seems more convenient?

Reply:
Ok, I didn't know you're using this elsewhere.